tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -220,147 +145,172 @@ def __call__(self, op):
     return device_name
 
 
-def _create_device_setter(is_cpu_ps, worker, num_gpus):
+def _create_device_setter(avg_on_gpu, worker, num_gpus):
```

Comment:
While silly, although not always silly you can put the variables on the GPUs and do the ops on CPU.  When you do replicated variables that is totally an option and in some amusing cases faster.